pkg/target/target.go

```diff
@@ -278,7 +278,21 @@ func addClusterLabels(opts *fleet.BundleDeploymentOptions, labels map[string]str
 		return
 	}
 
+	prefix := "global.fleet.clusterLabels."
+
+	for key, val := range opts.Helm.Values.Data {
+		valStr, ok := val.(string)
+		if ok && strings.HasPrefix(valStr, prefix) {
+			label := strings.TrimPrefix(valStr, prefix)
+			labelVal, labelPresent := clusterLabels[label]
+			if labelPresent {
+				opts.Helm.Values.Data[key] = labelVal
```

Comment:
Are we essentially reformatting the same data that's already been passed in? A bit confused here.

Reply:
Resolved from discussion.